BackofficeBundle/Form/DataTransformer/ChoiceArrayToStringTransformer.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace OpenOrchestra\BackofficeBundle\Form\DataTransformer;
+
+use Symfony\Component\Form\DataTransformerInterface;
+
+/**
+ * Class ChoiceArrayToStringTransformer
+ */
+class ChoiceArrayToStringTransformer implements DataTransformerInterface
+{
+    /**
+     * @param array|string $dataChoice
+     *
+     * @return string
+     */
+    public function transform($dataChoice)
+    {
+        if (is_scalar($dataChoice)) {
+            return (string) $dataChoice;
+        } elseif (is_array($dataChoice) && $dataChoice !== array()) {
+            return current($dataChoice);
+        } else {
+            return "";
+        }
+    }
+
+    /**
+     * @param string $dataChoice
+     *
+     * @return string
+     */
+    public function reverseTransform($dataChoice)
+    {
+        return $dataChoice;
```

Comment:
ok after talking about this, it's not a standard "view" <-> "normalized data" <-> "stored data" transformation, but a specific "old stored data format" <-> "new stored data format" so a reverse transform is not appropriate in this context